orc8r/cloud/docker/controller/Dockerfile

```diff
@@ -1,7 +1,9 @@
 # ------------------------------------------------------------------------------
 # Base: for tests, precommit, codegen, etc.
 # ------------------------------------------------------------------------------
-FROM ubuntu:xenial as base
+ARG PLATFORM=linux/amd64
+
+FROM --platform=$PLATFORM ubuntu:xenial as base
```

Comment:
Do we have any plans to upgrade from **xenial** to **focal**?

Reply:
You are right @Neudrino. LGTM.

Reply:
this PR just unblocks current M1 based builds. There is probably cleaner long term solution, but - I'd live it to the owners to figure out.

Reply:
@emakeev I have enabled Auto-merge on this PR. Just rebase it with the master so all tests can pass.